.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{vue,html,css,js,json,yml,ts,ftl}]
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Comment:
had to setup a new dev env, so copied over the editorconfig from services-ui, and added the ftl templates too.